emsdk.py

```diff
@@ -2990,8 +2946,7 @@ def print_tools(t):
       rmfile(sdk_path(EMSDK_SET_ENV))
     return 0
   elif cmd == 'update-tags':
-    update_tags()
-    return 0
+    exit_with_error('`update-tags` is not longer needed.  To install the latest tot release just run `install tot`')
```

Comment:
How about making this a warning, and saying it's a no-op? I'd rather not break users.

Reply:
OK sounds good.